### PR TITLE
Added analysis for location.region_concept_id

### PR DESCRIPTION
**Pull request by pavgra**
To support https://github.com/OHDSI/CommonDataModel/issues/252

### CODE REVIEW

File: R/Achilles.R [hunk 1]
```diff
@@ -51,6 +51,7 @@
 #' @param runHeel                          Boolean to determine if Achilles Heel data quality reporting will be produced based on the summary statistics.  Default = TRUE
 #' @param validateSchema                   Boolean to determine if CDM Schema Validation should be run. Default = FALSE
 #' @param runCostAnalysis                  Boolean to determine if cost analysis should be run. Note: only works on v5.1+ style cost tables.
+#' @param runLocationAnalysis              Boolean to determine if location analysis should be run. Note: only works on v6.1+ style location tables.
```
Comment (konstjar) [inline R/Achilles.R]: Can we add this parameter in Docker too?

**Comment by alondhe** [comment]
@pavgra is this a CDM v6 feature? We don't yet support v6 in Achilles (though there is a very untested branch). 

At Janssen, we are planning a Synthea v6 to help developers test against, it is due out by June. For Achilles, we are planning v6 support in our v1.8.0 release, which is due out in June as well.

**Comment by pavgra** [comment]
Ok, @alondhe , I will rebase the PR

**Comment by alondhe** [comment]
@pavgra -- Apologies for the delay. I've restarted V6 work in the branch cdm_v6_2020. Can you create a new PR for that branch? That branch has achilles analyses that work against v6; but cost analyses are in progress.

**Pull request closed without merging**